zerver/lib/events.py

```diff
@@ -273,6 +278,9 @@ def fetch_initial_state_data(user_profile: UserProfile,
                                         'config': load_bot_config_template(bot.name)})
         state['realm_embedded_bots'] = realm_embedded_bots
 
+    if want('recent_conversations'):
```

Comment:
Rename this to `recent_private_conversations`.

See https://github.com/zulip/zulip-mobile/issues/3133#issuecomment-475115028